framework/test/unit/engine/consensus/certificate_generation/commit_pool.spec.ts

```diff
@@ -1003,6 +1081,30 @@ describe('CommitPool', () => {
 			expect(isCommitVerified).toBeFalse();
 		});
 
+		it('should return true when aggregationBits empty and certificateSignature is empty and aggregateCommit.height == getBFTHeights().maxHeightCertified', async () => {
```

Comment:
Since `aggregationBits`, `certificateSignature` and `height` are in `aggregateCommit`. How about:
```
should return true when aggregationBits === EMPTY_BYTES and certificateSignature === EMPTY_BYTES and height === getBFTHeights().maxHeightCertified in aggregateCommit
```